test/object_store/test_basic.py

```diff
@@ -18,6 +18,7 @@
 def get_scylla_with_s3_cmd(ssl, s3_server):
     '''return a function which in turn returns the command for running scylla'''
     scylla = run.find_scylla()
+    print('====', os.getenv('SCYLLA'))
```

Comment:
But patch title says it "pass" SCYLLA, not "print"

Reply:
I think you forgot to add the patch to test.py. This is just the debugging printout.
By the way, even for a debugging printout, please make it clearer, maybe "Scylla is", not "====".

Reply:
sorry, i cherry-picked the wrong chunk of code.